doc/usage.rst

```diff
@@ -237,63 +237,65 @@ All command line options
+  | win:autoScreensaver        |       | no                     | Prevent the screensaver from starting when guest requests it         |
+  +----------------------------+-------+------------------------+----------------------------------------------------------------------+
+  | win:alerts                 | -q    | yes                    | Show on screen alert messages                                        |
+  +----------------------------+-------+------------------------+----------------------------------------------------------------------+
+  | win:quickSplash            |       | no                     | Skip fading out the splash screen when a connection is established   |
+  +----------------------------+-------+------------------------+----------------------------------------------------------------------+
+  | win:overlayDimsDesktop     |       | no                     | Dim the desktop when in interactive overlay mode                     |
+  +----------------------------+-------+------------------------+----------------------------------------------------------------------+
+  | win:rotate                 |       | 0                      | Rotate the displayed image (0, 90, 180, 270)                         |
+  +----------------------------+-------+------------------------+----------------------------------------------------------------------+
+  | win:uiFont                 |       | DejaVu Sans Mono       | The font to use when rendering on-screen UI                          |
+  +----------------------------+-------+------------------------+----------------------------------------------------------------------+
+  | win:uiSize                 |       | 14                     | The font size to use when rendering on-screen UI                     |
+  +----------------------------+-------+------------------------+----------------------------------------------------------------------+
+  | win:jitRender              |       | no                     | Enable just-in-time rendering                                        |
+  +----------------------------+-------+------------------------+----------------------------------------------------------------------+
+  | win:requestActivation      |       | yes                    | Request activation when attention is needed                          |
+  +----------------------------+-------+------------------------+----------------------------------------------------------------------+
+  | win:showFPS                | -k    | no                     | Enable the FPS & UPS display                                         |
+  +----------------------------+-------+------------------------+----------------------------------------------------------------------+
+  | win:disableWaitingMessage  |       | no                     | Disables the confirmation message for a cleaner UI                                      |
```

Comment:
The formatting here is incorrect. Please use `./looking-glass-client --rst-help` to generate the help for this documentation.

Reply:
Should be fixed now, sorry.